10823 update slack installation doc (#10940)
RasaHQ/rasa: docs/docs/connectors/slack.mdx

   If you are running locally, make sure ngrok (or another tool to retrieve a public
   url) is running as well.

> 2. To send messages directly to your bot using the slack UI, head to **App Home**,
>    scroll to bottom and turn on checkbox for
>    `Allow users to send Slash commands and messages from the messages tab.`
> 
>    You might have to quit the Slack app and re-open it before this takes affect
> 
>    <Image img={appHomeImg} caption="Messages Tab" alt="Allow users to send Slash commands and messages from the messages tab" max-width="500px"/>
> 
> 3. Configure the webhook by heading to **Event Subscriptions** and

Reviewer: ```suggestion
2. To send messages directly to your bot using the slack UI, head to **App Home**,
   scroll to the bottom and select the checkbox for
   `Allow users to send Slash commands and messages from the messages tab.`

   You might have to quit the Slack app and re-open it before your changes take effect.

   <Image img={appHomeImg} caption="Messages Tab" alt="Allow users to send Slash commands and messages from the messages tab" max-width="500px"/>

3. Configure the webhook by heading to **Event Subscriptions** and
```
